core/src/test/java/io/grpc/internal/DnsNameResolverTest.java

```diff
@@ -116,6 +117,7 @@ public void uncaughtException(Thread t, Throwable e) {
       .setProxyDetector(GrpcUtil.DEFAULT_PROXY_DETECTOR)
       .setSynchronizationContext(syncContext)
       .setServiceConfigParser(mock(ServiceConfigParser.class))
+      .setScheduledExecutorService(mock(ScheduledExecutorService.class))
```

Comment:
This one too. Is it necessary to set it? Can we leave it unset everywhere it is unused?

Reply:
Sure. It doesn't matter much. I set it just for completeness.

Reply:
I'm mainly requesting it because it makes the tests more bug-prone. As it is now if the code starts using the scheduled executor service it will throw and then it is clear a FakeClock needs to be injected. With the mock it just silently doesn't do anything.